packages/create-vite/build.config.ts

```diff
@@ -23,7 +25,7 @@ export default defineBuildConfig({
       if (!options.plugins) {
         options.plugins = []
       }
-      options.plugins.push(
+      ;(options.plugins as Plugin[]).push(
         licensePlugin(
           path.resolve(__dirname, './LICENSE'),
           'create-vite license',
```

Comment:
Because rollup now supports nested plugins, I think we can remove this `if (!options.plugins)` + `options.plugins` and just write:
```ts
      options.plugins = [
        options.plugins,
        licensePlugin(
          path.resolve(__dirname, './LICENSE'),
          'create-vite license',
          'create-vite',
        ),
      ]
```